share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
+    set -l _matched (string match -r "^ +([^ ]*) *([^ ]?.*)" $task)
+
+    if test -n "$_matched"
+        set -l _task_name $_matched[2]
+        set -l _task_desc $_matched[3]
+        test -n "$task_desc"; or set -l _task_desc "No description"
+        complete -f -c fab -a "$_task_name" -d "$_task_desc"
+    end
+end
+
+# options
+complete -c fab -s h -l help -d "show this help message and exit"
+complete -x -c fab -s d -l display -a "(fab --shortlist)" -d "print detailed info about command NAME"
+complete -x -c fab -s F -l list-format -a "short normal nested" -d "formats --list, choices: short, normal, nested"
+complete -c fab -s I -l initial-password-prompt -d "Force password prompt up-front"
+complete -c fab -l initial-sudo-password-prompt -d "Force sudo password prompt up-front"
+complete -c fab -s l -l list -d "print list of possible commands and exit"
+complete -x -c fab -l set -d "comma separated KEY=VALUE pairs to set Fab env vars"
+complete -c fab -l shortlist -d "alias for -F short --list"
+complete -c fab -s V -l version -d "show program's version number and exit"
+complete -c fab -s a -l no_agent -d "don't user the running SSH agent"
+complete -c fab -s A -l forward-agent -d "forward local agent to remote end"
+complete -c fab -l abort-on-prompts -d "abort instead of prompting (for password, host, etc)"
+complete -x -c fab -s c -l config -d "specify location of config file to use"
+complete -c fab -l colorize-errors -d "Color error output"
+complete -c fab -s D -l disable-known-host -d "do not load user known_hosts file"
+complete -c fab -s e -l eagerly-disconnect -d "disconnect from hosts as soon as possible"
+complete -x -c fab -s f -l fabfile -d "python module file to import, e.g. '../other.py'"
+complete -x -c fab -s g -l gateway -a "
+(__fish_print_hostnames)
+(
+	echo (commandline -ct)|sed -ne 's/\(.*@\).*/\1/p'
```

Comment:
This should be rewritten to use `string replace`.

Something like `commandline -ct | string replace -r '(.*@).*' '$1'` (the `echo` is superfluous).